packages/browser.smash

```diff
@@ -152,6 +152,10 @@
         await browser.driver.actions({bridge: true}).move({origin: await $(element, true)}).perform();
     }
 
+    * Move mouse from element {{element}} by {{x}}, {{y}} {
```

Comment:
Maybe remove the word "element" as it is redundant :)

Reply:
Oops!